### ISSUE TITLE
Address Grebel suggestions

### ISSUE THREAD

**Issue by andycasey** [AnnieJumpCannon/RAVE#31]
- [x] ~~In your Introduction on p. 2, 1st paragraph, it would be nice if you
  could also cite some of the RAVE studies of the structure, kinematics,
  and chemistry of the Milky Way, e.g., Pasetto et al. 2012a, A&A, 547,
  70; Pasetto et al. 2012b, A&A, 547, 71; Boeche et al. 2013, A&A, 559,
  59; Boeche et al. 2014, A&A, 568, 71.~~
- [x] On p. 2 in par. 3, when you mention RAVE for the first time, Steinmetz
  et al. (2006, AJ, 132, 1645) should be cited.
  In the footnote, it would be also useful to state which RAVE data
  release(s) you are using and to cite the corresponding paper(s).
- [x] On p. 4 in par. 2, line 2 you mention the RAVE data releases - why
  not cite them here as well to give credit?  (Kunder et al. 2016
  for DR5, Kordopatis et al. 2013 for DR4, Siebert et al. 2011 for
  DR3, Zwitter et al. 2008 for DR2, and Steinmetz et al. 2006 for DR1).
  In line 4 for individual abundances it would be nice to also cite
  Boeche et al. (2011; the chemical catalog paper).
  You write that in the current paper DR5-processed spectra were used,
  yet later comparisons are made with DR4.  Why not compare to the
  (improved) output of DR5?
- [x] On p. 7, where you added the comment that the sample was supplemented
  with known metal-poor stars in RAVE:  You are probably referring to
  Fulbright et al. (2010) and to Ruchti et al. (2011a, 2011b) here and
  plan to cite and discuss them?
- [x] ~~How do you deal with those RAVE stars that were identified as double-
  lined (Matijevic et al. 2010) or single-lined (Matijevic et al.
  2011) binaries or as stars with chromospheric emission (Zerjal et
  al. 2013)?  Or, turning this question around, what does the Cannon
  make of these stars?~~ **[out of scope; we encourage use of the morphological flags and already talk at length about projection effects due to unusual spectra]**
- [x] ~~P. 9, par. 2, morphological classifications:  Did you consider the
  morphological classifications of RAVE stars by Matijevic et al.
  (2012) here?  If yes, it would be good to mention that; if no perhaps
  briefly explain why this information was not used.~~ **[we weren't blessed with a large training set, and the morphological classifications can produce a very conservative training set by culling some stars that are normal]**
- [x] P. 10/13:  Wouldn't it be a useful test to check which results the
  Cannon found for RAVE stars with repeat observations, i.e., to
  compare the results for the first spectrum with those of the second?
  I understand that you only stacked them, but did not compare the
  results for individual spectra of the same object?
- [x] ~~P. 11, par. 3, spurious results:  Can these be traced to the known
  spectroscopic binaries or other stars with previously identified
  peculiarities in RAVE (i.e., from the Matijevic et al. papers or
  Zerjal et al.)?~~ **[no longer relevant in most recent draft]**
- [x] ~~Section 4.2.1:  Why not also compare with DR5?~~
- [x] ~~Section 4.2.2:  Wouldn't a comparison with the Fulbright/Ruchti
  high-resolution spectra of RAVE stars be useful as well?~~ **[incorporated as part of the Kordopatis calibration sample, and this comparison is deceptive because some of these stars were in the training set]**
- [x] P. 16, par. 4:  How about also citing Kunder et al. (2014)?
- [x] P. 19, par. 1:  As far as I can see, this is the first time that
  Matijevic et al.'s morphological classifications are mentioned -
  as suggested above I would recommend to introduce them already
  much earlier (and specifically for chromospheric emission also
  Zerjal et al. 2013).
- [x] P. 19, par. 3, "presented the largest collection...":  How about:
  "We presented a recalculation of stellar abundances based on the
  largest collection of such measurements in RAVE DR5.  These stars
  overlap with Gaia... and thus...".  (or something along these lines)
  In other words, I would state that DR5 is the largest collection
  and that you are now presenting what likely is a more precise
  recalculation.
